helm-charts/support/values.schema.yaml

```diff
@@ -70,12 +72,13 @@ properties:
               type: string
               description: |
                 Domain to redirect to
-            code:
-              type: integer
+            type:
+              type: string
+              enum: ["redirect", "permanent"]
               description: |
-                HTTP Status code to use for the redirect.
+                Relates to the HTTP Status code to use for the redirect.
 
-                302 (Temporary Redirect) is used if none is specified
+                Specify "redirect" for 302 (default), or "permanent" for 301.
```

Comment:
The `code` configuration wasn't used, so I switched it to allowing users to specify `type` as that was easier to implement using the `rewrite` nginx config.